### PR TITLE
feat(proposals) v2 cleanup

### LINKED ISSUE #2061: Proposals: v2
## Design
https://xd.adobe.com/view/b3579d9c-c8b7-43c1-9fe1-a5a11f92e3ad-e604/

## Passing Criteria

- [x] as a DAO user, I want to have the ability to see staging proposals in the list view
- [ ] as a DAO user, I want to have the ability to see the proposal history
- [ ] as a DAO user, I want to have the ability to see the circle name on the proposal card
- [ ] as a DAO user, I want to have the ability to see the archetype name on the proposal card

### PR DESCRIPTION
**Pull request by evgenibir**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

Proposals: v2 #2061 

### ✅ Checklist

- Removed view selector from filter widget and added titles with counters
- Improved filter widget
- Improved proposal card styles in proposals page
- Added proposal history widget

### 🙈 Screenshots
image
image

close #2061

### CODE REVIEW

File: src/pages/proposals/ProposalList.vue [hunk 1]
```diff
@@ -475,12 +497,41 @@ q-page.page-proposals
...
+      :toggleLabel="'Staging Proposals'"
       :toggle.sync="showStagedProposals",
       :toggleDefault="true"
       :showToggle="true",
       :style="mobileFilterStyles"
+      :filterTitle="'Search proposals'"
+      :showViewSelector="false"
       )
+      .col
+        base-placeholder.q-mr-sm(v-if="!filteredProposals.length && !filteredStagedProposals.length && !$apollo.loading" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
```
Comment (Alex-IS) [inline src/pages/proposals/ProposalList.vue:509]: @evgenibir It is not really readable like this. Put each property in a new line, like:
```vue
...
title="No Proposals"
subtitle="Your organization has not created..."
...
```

Comment (evgenibir) [reply src/pages/proposals/ProposalList.vue]: fixed

File: src/pages/proposals/ProposalList.vue [hunk 2]
```diff
@@ -475,12 +497,46 @@ q-page.page-proposals
...
+      :filterTitle="'Search proposals'"
+      :showViewSelector="false"
       )
+      .col
+        base-placeholder.q-mr-sm(
+          v-if="!filteredProposals.length && !filteredStagedProposals.length && !$apollo.loading"
+          title= "No Proposals"
+          subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
+          icon= "fas fa-file-medical"
+          :actionButtons="[{label: 'Create a new Proposal', color: 'primary', onClick: () => $router.push(`/${this.daoSettings.url}/proposals/create`), disable: !isMember, disableTooltip: 'You must be a member'}]"
```
Comment (Alex-IS) [inline src/pages/proposals/ProposalList.vue:514]: @evgenibir  would be nicer to have `onClick: handleCreateNewProposal` and above in JS to have method `handleCreateNewProposal`. This inlined method can become messy pretty easily

**Comment by Alex-IS** [comment]
@evgenibir Also make sure that there are no conflicts. Thanks

File: src/components/filters/filter-widget.vue [hunk 3]
```diff
@@ -160,6 +160,29 @@ widget(title="Filters")
       .row.q-my-md(v-if="filters")
           .h-b2.q-mb-sm {{ chipsFiltersLabel }}
           chips(:tags="filterTags" clickable @click-tag="toggleFilter" )
+      .row.items-center.justify-between.q-py-sm(v-if="showViewSelector")
+        .h-b2 {{ viewSelectorLabel }}
+        .btn-container
+          q-btn.q-mr-xxs(
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/components/filters/filter-widget.vue:166]: @Alex-IS didn't quite understand the problem. **Rounded** and **unelevated** are the parameters of the q-btn component, sometimes we use them, and sometimes not, depending on the design of a particular page

File: src/pages/proposals/ProposalList.vue [hunk 4]
```diff
@@ -455,17 +476,18 @@ q-page.page-proposals
       :viewSelectorLabel="'View'",
       :chipsFiltersLabel="'Proposal types'",
       :filters.sync="filters"
-      :toggleLabel="'Show Staging Proposals'"
+      :toggleLabel="'Staging Proposals'"
```
[the remark this answers is not in the document]
Comment (evgenibir) [reply src/pages/proposals/ProposalList.vue]: fixed

**Comment by evgenibir** [comment]
> @evgenibir Also make sure that there are no conflicts. Thanks

Merged develop

**Pull request merged**